zstacklib/zstacklib/test/test_safe_delete.py

```
@@ -0,0 +1,299 @@
+_ns['logger'] = _logging.getLogger('test_safe_delete')
+_ns['_string_types'] = (str,) if sys.version_info[0] >= 3 else (str, unicode)
+
+
+# We also need rm_dir_force and rm_file_force
+def _rm_dir_force(path):
+    shutil.rmtree(path)
+
+def _rm_file_force(path):
+    try:
+        os.remove(path)
+    except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)